### PR TITLE
Update Ubee component documentation

### PR DESCRIPTION
**Pull request by mzdrale**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22450

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/ubee.markdown [hunk 1]
```diff
@@ -10,8 +10,6 @@ footer: true
 logo: ubee.png
 ha_category: Presence Detection
 ha_release: 0.89
-redirect_from:
- - /components/device_tracker.ubee/
```
Comment (MartinHjelmare) [inline source/_components/ubee.markdown]: We should keep this if there used to be to be a page at that place.

File: source/_components/ubee.markdown [hunk 2]
```diff
@@ -28,8 +28,12 @@ device_tracker:
 ```
 
 {% configuration %}
+model:
+  description: Ubee Router model, e.g. `EVW32C-0N`. If omitted, model will be detected automatically.
+  required: false
+  type: string
```
Comment (MartinHjelmare) [inline source/_components/ubee.markdown:35]: What's the default?

Comment (mzdrale) [reply source/_components/ubee.markdown:35]: Default is 'detect', to detect model automatically.

Comment (MartinHjelmare) [reply source/_components/ubee.markdown:35]: Let's add that for key `default`.

File: source/_components/ubee.markdown (hunk 1 above)
Comment (mzdrale) [reply source/_components/ubee.markdown]: Added it back.

**Review by MartinHjelmare — APPROVED** [review]
Looks good!

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**